README.md

```diff
@@ -188,6 +215,56 @@ Possible values of this variable:
 
 Note: If not set explicitly on registration, the display name is set to the hostname by default. It is not possible to automatically revert it to the hostname, but it can be set so manually.
 
+```yaml
+    rhc_insights:
+      file_redaction:
+        commands: []
+        files: []
+        components: []
+```
+
+Specify lists of commands, files, and components to obmit from output
```

Comment:
```suggestion
Specify lists of commands, files, and components to omit from output
```

Reply:
Fixed in commit [ff0199b](https://github.com/linux-system-roles/rhc/pull/177/commits/ff0199b1f95e5490a32844b1530f5b49f3a0167f)